RELEASE_NOTES.md

```diff
@@ -72,7 +72,8 @@ accompanying blog post.
   a smooth transition effect. The effect can be sped up or switched 
   off with preferences/miscellaneous/duration of ui transitions.
 
-- Large pixelpipe cache overhaul.
+- Large pixelpipe cache overhaul. Increased number of cachelines with
```

Comment:
Compared to what you did this seems a very small description. I've done the initial edit for remembering that we have something to write here. Do not hesitate to add more information. The hard and important work you did on this part may really need more advertising.

Reply:
I am fine here with the small addition just describing what has been done. The size of text indeed does not correspond with the amount of work, not so important. The user should just feel the responsiveness so a short text seems to be appropriate.